Cargo.toml

```diff
@@ -58,6 +58,7 @@ tokio = { version = "1.29.1", features = [
 tokio-rustls = "0.23.4"
 toml = "0.7.6"
 humansize = "2.1.3"
+colored = "2.0.4"
```

Comment:
Can we add colors in another PR? So we revisit all places and keep UI consistent?